### ISSUE TITLE
tab shortcut after the trigger is not working.

### ISSUE THREAD

**Issue by manojp1988** [xabikos/vscode-jasmine#9]
I am using vscode 1.13.0 with eclipse keymap bindings. In the typescript file, if I type 'it' and then tab key, is not generating the code snippets. I have to use ctrl+space and select 'it' from the drop down to generate the snippets.

Is it I am missing ?

**Comment by xabikos** [comment]
@manojp1988 I am terrible sorry for the really late reply. Do you have installed other snippets? Does this happening only with Jasmine snippets?

**Comment by manojp1988** [comment]
Yes this is happening only with jasmine snippets. I have installed other
plug-in for jasmine which is working correctly. In that plug in, I have to
type 'testd' to generate describe block.

On Sep 26, 2017 8:59 PM, "Charalampos Karypidis" <notifications@github.com>
wrote:

@manojp1988 <https://github.com/manojp1988> I am terrible sorry for the
really late reply. Do you have installed other snippets? Does this
happening only with Jasmine snippets?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/xabikos/vscode-jasmine/issues/9#issuecomment-332236622>,
or mute the thread

**Comment by lavenpillay** [comment]
Hi, 
Any update on this ?

**Comment by jQIX** [comment]
This worked for me: set "editor.snippetSuggestions" to "top" (Controls whether snippets are shown with other suggestions and how they are sorted.)

**Comment by vegetabill** [comment]
@jQIX Thanks for that tip! It worked for me. Any idea why this is the case? I've used other snippets for mocha, including my own, and they worked without having to change settings.

**Comment by Cito** [comment]
Came here to report the same issue. The problem exists only for `it`, not for `desc` and others.
